cuckoo/cuckoo_main.py

```diff
@@ -760,9 +760,11 @@ def query_machines(self) -> None:
                 number_of_unavailable_hosts += 1
                 continue
             except requests.ConnectionError:
-                raise Exception(f"Unable to reach the Cuckoo nest ({host['ip']}) while trying to query machines. "
-                                f"Be sure to checkout the README and ensure that you have a Cuckoo nest setup outside "
-                                f"of Assemblyline first before running the service.")
+                self.log.error(f"Unable to reach the Cuckoo nest ({host['ip']}) while trying to query machines. "
+                               f"Be sure to checkout the README and ensure that you have a Cuckoo nest setup outside "
+                               f"of Assemblyline first before running the service.")
+                number_of_unavailable_hosts += 1
+                continue
```

Comment:
Shouldn't the "machines" key be deleted from the host to indicate that no machines are currently connectable?

Reply:
removing unresponsive hosts